lib/load.php

```diff
@@ -102,6 +102,7 @@ function gutenberg_is_experiment_enabled( $name ) {
 require __DIR__ . '/experimental/synchronization.php';
 require __DIR__ . '/experimental/script-modules.php';
 require __DIR__ . '/experimental/posts/load.php';
+require __DIR__ . '/experimental/workflow-palette.php';
```

Comment:
Could we wrap the require inside "if ( gutenberg_is_experiment_enabled(" like in some other cases on this file?